VersionControlUI/CMakeLists.txt

```diff
@@ -9,10 +9,16 @@ add_library(VersionControlUI SHARED
 	src/versioncontrolui_api.h
 	src/VersionControlUIPlugin.h
 	src/DiffManager.h
+	src/DiffComparisonPair.h
```

Comment:
you should put node files in a `nodes` subdirectory and item files and styles in an `items` subdirectory